### PR TITLE
Debugging hound issues

### PR DESCRIPTION
**Pull request by tag6071**
Trying to make sure issues pointed out by hound are fixed @sule kindly help review thanks

### CODE REVIEW

File: template/css/main.scss [hunk 1]
```diff
@@ -165,8 +161,15 @@ $teal-variant: #00a8a8;
       font-size: 2em;
     }
   }
+
+  li>a>i {
```
[the remark this answers is not in the document]
Comment (osule) [reply template/css/main.scss]: You should probably use a class on the links with this icon.  You can then apply this in your markup.

.margin-0 {
   margin: 0;
}

[the remark this answers is not in the document]
Comment (tag6071) [reply template/css/main.scss]: I did something like that earlier, but it is being overridden by **materialize css** styling and **important** is not accepted either...Is it safe or legit to disable the rule for this line @osule ?

[the remark this answers is not in the document]
Comment (osule) [reply template/css/main.scss]: What's the materialize selector overriding this?

[the remark this answers is not in the document]
Comment (tag6071) [reply template/css/main.scss]: here `.side-nav li>a>i`

[the remark this answers is not in the document]
Comment (osule) [reply template/css/main.scss]: See https://specificity.keegan.st/

[the remark this answers is not in the document]
Comment (osule) [reply template/css/main.scss]: A more specific selector is `.icon__link .margin-0`.

Here you need to add an `icon__link` class to markup for `a` tags containing the icon and add a `margin-0` to your icons.
```
.icon__link .margin-0 {
    margin: 0;
}
```

[the remark this answers is not in the document]
Comment (tag6071) [reply template/css/main.scss]: Yes thanks, just did saying **4** violations now and its not showing me those violations.

**Comment by tag6071** [comment]
kindly review this commit above @osule thanks

File: client/public/css/main.css [hunk 2]
```diff
@@ -75,9 +75,14 @@
...
   background-size: cover; }
+  .shelf-page nav {
+    background-color: #fff; }
+    .shelf-page nav ul > li > a {
+      color: #666;
+      font-size: 16px; }
 
 .admin-page {
-  background: none !important; }
+  background: none; }
```
Comment (osule) [inline client/public/css/main.css]: What part of the background are you styling here?
```
background-image: none
```

**Review by osule — APPROVED** [review]

**Comment by tag6071** [comment]
@osule please do look at this commit, i did some refactorings.

**Pull request merged**